tests/unit/openpype/plugins/publish/test_validate_sequence_frames.py

```diff
@@ -0,0 +1,144 @@
+
+
+"""Test Publish_plugins pipeline publish modul, tests API methods
+
+    File:
+        creates temporary directory and downloads .zip file from GDrive
+        unzips .zip file
+        uses content of .zip file (MongoDB's dumps) to import to new databases
+        with use of 'monkeypatch_session' modifies required env vars
+            temporarily
+        runs battery of tests checking that site operation for Sync Server
+            module are working
+        removes temporary folder
+        removes temporary databases (?)
+"""
+import pytest
+import logging
+
+from pyblish.api import Instance as PyblishInstance
+
+from tests.lib.testing_classes import BaseTest
+from openpype.plugins.publish.validate_sequence_frames import (
```

Comment:
Are we sure this works, the folder doesn't have an `__init__.py` right?

Reply:
It is working for me in Pycharm, wanted to tested directly from command line, but didn't get to it.

Reply:
Works for me even from command line with `start.py`

Reply:
if you specify full "path" with dots, subfolders don't have to contain `__init__.py`. (`import openpype.plugins.publish.validate_sequence_frames` would work but `from openpype.plugins.publish import validate_sequence_frames` would not)